addons/featured-lists/userscript.js

```diff
@@ -0,0 +1,59 @@
+export default async function ({ addon, console }) {
+  let modSplashes;
+  changeRowsOrder();
+
+  addon.self.addEventListener("disabled", function () {
+    modSplashes = document.querySelectorAll(".mod-splash");
+    modSplashes[0].style.display = "block"; // If user is not logged in
+    modSplashes[1].style.display = "block";
```

Comment:
Could this be assigned to a second variable based on if the user is logged in instead of just running both since it's done like this elsewhere too?